src/features/Status/StatusCTAContainer.tsx

```diff
@@ -57,30 +74,58 @@ class StatusCTAContainer extends React.Component<IStatusCTAContainerProps, IStat
     }
   };
 
+  public modifyHacker = async (hacker: IHacker) => {
+    const newHacker = { ...hacker };
+    const { timeZone, country, city } = this.state;
+    newHacker.application.location = {
+      timeZone,
+      country,
+      city,
+    };
+    return newHacker;
+  };
+
   public render() {
+    const { timeZone, country, city } = this.state;
     return (
-      <Flex flex={1} flexDirection="column" justifyContent="center" alignItems="center" p="6px 30px 96px 30px">
+      <Flex
+        flex={1}
+        flexDirection="column"
+        justifyContent="center"
+        alignItems="center"
+        p="6px 30px 96px 30px"
+      >
         {this.props.confirmed && this.props.account ? (
           <StatusCTA
             status={this.state.status}
             firstName={this.props.account.firstName}
             settings={this.state.settings}
-            onClickConfirm={this.confirmStatus}
+            onClickConfirm={async () =>
+              this.setState({ isLocationModalOpen: true })
+            }
             // tslint:disable-next-line: jsx-no-lambda
-            onClickWithdraw={() => this.setState({ isModalOpen: true })}
+            onClickWithdraw={() => this.setState({ isWithdrawModalOpen: true })}
           />
         ) : (
           <ConfirmationEmailSentComponent />
         )}
+        <StatusCTALocationModal
+          isModalOpen={this.state.isLocationModalOpen}
+          onCanceled={() => this.setState({ isLocationModalOpen: false })}
+          onConfirmed={this.confirmStatus}
+          timeZone={timeZone}
+          country={country}
+          city={city}
+          handleChangeTimeZone={this.handleChangeTimeZone}
+          handleChangeCountry={this.handleChangeCountry}
+          handleChangeCity={console.log('test')}
+        />
         <ConfirmModal
-          isOpen={this.state.isModalOpen}
+          isOpen={this.state.isWithdrawModalOpen}
           // tslint:disable-next-line: jsx-no-lambda
-          onCanceled={() => this.setState({ isModalOpen: false })}
+          onCanceled={() => this.setState({ isWithdrawModalOpen: false })}
           // tslint:disable-next-line: jsx-no-lambda
-          onConfirmed={() => {
-            this.withdrawStatus();
```

Comment:
Why was the withdrawStatus function removed?

Reply:
error on my part 😓